python/ray/data/_internal/aggregate.py

```diff
@@ -363,3 +363,51 @@ def percentile(input_values, key: Optional[Callable[[Any], Any]] = None):
             finalize=_null_wrap_finalize(percentile),
             name=(self._rs_name),
         )
+
+
+class Unique(_AggregateOnKeyBase):
+    """Defines unique aggregation."""
+
+    def __init__(
+        self,
+        on: Optional[str] = None,
+        ignore_nulls: bool = False,
+        alias_name: Optional[str] = None,
```

Comment:
this `alias_name` arg doesn't seem to be used?

Reply:
it's used a few lines below to update `self._rs_name` (or AggregateFn's name). It follows other aggregations in the same file.